assets/src/components/parts/janus-mcq/schema.ts

```diff
@@ -78,10 +79,27 @@ export const adaptivitySchema = {
   selectedChoicesText: CapiVariableTypes.ARRAY,
 };
 
+export const getFormattedExpression = (part: any, owner: any): Expression[] => {
+  const brokenExpressions: Expression[] = [];
+  part.custom.mcqItems.forEach((element: any) => {
```

Comment:
This could be refactored as a `map` instead of `forEach`.

`const brokenExpressions: Expression[] = part.custom.mcqItems.map`